packages/framework/src/Component/UploadedFile/UploadedFileRepository.php

```diff
@@ -81,4 +81,28 @@ public function getById(int $uploadedFileId): UploadedFile
 
         return $uploadedFile;
     }
+
+    /**
+     * @param int $uploadedFileId
+     * @param string $uploadedFileSlug
+     * @param string $uploadedFileExtension
+     * @return \Shopsys\FrameworkBundle\Component\UploadedFile\UploadedFile
+     */
+    public function getByIdSlugAndExtension(int $uploadedFileId, string $uploadedFileSlug, string $uploadedFileExtension): UploadedFile
+    {
+        $uploadedFile = $this->getUploadedFileRepository()->findOneBy(
+            [
```

Comment:
Just a tiny nitpick. I would left this brace on previous line. Readibility wiil stay the same IMO.

Reply:
updated

Reply:
Actually you updated different occurency 😄

Reply:
Well that means I am rewriting it back and keeping, so it is united

Reply:
Why not to rewrite the second one as well?